application/controllers/api/ApiController.php

```diff
@@ -0,0 +1,83 @@
+<?php
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+/**
+ * Classe ApiController.
+ * 
+ * @extends REST_Controller
+ */
+require(APPPATH.'/libraries/REST_Controller.php');
```

Comment:
Sugestão: Acho melhor tentar colocar o REST_Controller no autoload do composer, pra não ter que colocar o require em todos os arquivos.

Reply:
Não pensei em colocar pq só será usado na API, e quem não for usar a api não precisa carregar o RET_Controller.
Realmente recomenda colocar no autoload?

Reply:
O custo de carregar no autoload é minimo e o benefício compensa.

Reply:
ok